exporter/exportertest/sink_exporter_test.go

```diff
@@ -51,16 +49,12 @@ func TestSinkMetricsExporter(t *testing.T) {
 	}
 	want := make([]consumerdata.MetricsData, 0, 7)
 	for i := 0; i < 7; i++ {
-		if err := sink.ConsumeMetricsData(context.Background(), md); err != nil {
-			t.Fatalf("Wanted nil got error")
-		}
+		err := sink.ConsumeMetricsData(context.Background(), md)
+		require.Nil(t, err)
 		want = append(want, md)
 	}
 	got := sink.AllMetrics()
-	if !reflect.DeepEqual(got, want) {
-		t.Errorf("Mismatches responses\nGot:\n\t%v\nWant:\n\t%v\n", got, want)
-	}
-	if sink.Name() != "sink_metrics" {
-		t.Errorf("Wanted sink_metrics got %s", sink.Name())
-	}
+	assert.True(t, reflect.DeepEqual(got, want))
```

Comment:
Same here: use `assert.Equal(t, want, got)`

Reply:
@pjanotti thanks for the guidance.